build/Build.cs

```diff
@@ -118,63 +101,63 @@ public Build()
+                                if (filesContainingForbiddenWords.Any())
+                                    throw new Exception("Found forbidden words in the following files, "
+                                                        + "please clean them up:\r\n"
+                                                        + string.Join("\r\n", filesContainingForbiddenWords));
+                            });
 
         Target Clean =>
             _ => _.Executes(() =>
-            {
-                SourceDirectory.GlobDirectories("**/bin", "**/obj", "**/TestResults").ForEach(DeleteDirectory);
-                EnsureCleanDirectory(ArtifactsDirectory);
-                EnsureCleanDirectory(PublishDirectory);
-            });
+                            {
+                                SourceDirectory.GlobDirectories("**/bin", "**/obj", "**/TestResults").ForEach(DeleteDirectory);
+                                EnsureCleanDirectory(ArtifactsDirectory);
+                                EnsureCleanDirectory(PublishDirectory);
+                            });
 
         Target Restore =>
             _ => _.DependsOn(Clean)
                   .Executes(() =>
-                  {
-                      var localRuntime = FixedRuntimes.Windows;
+                            {
+                                var localRuntime = FixedRuntimes.Windows;
 
-                      if (!OperatingSystem.IsWindows())
-                          localRuntime = FixedRuntimes.Linux;
+                                if (!OperatingSystem.IsWindows())
+                                    localRuntime = FixedRuntimes.Linux;
 
-                      DotNetRestore(_ => _.SetProjectFile(Solution)
-                                          .SetRuntime(localRuntime)
-                                          .SetProperty("DisableImplicitNuGetFallbackFolder", true));
-                  });
+                                DotNetRestore(_ => _.SetProjectFile(Solution)
+                                                    .SetRuntime(localRuntime)
+                                                    .SetProperty("DisableImplicitNuGetFallbackFolder", true));
+                            });
 
         Target Compile =>
-            _ => _.DependsOn(CheckForbiddenWords)
-                  .DependsOn(Restore)
-                  .Executes(() =>
-                  {
-                      Log.Information("Compiling Calamari v{CalamariVersion}", NugetVersion.Value);
-
-                      DotNetBuild(_ => _.SetProjectFile(Solution)
-                                        .SetConfiguration(Configuration)
-                                        .SetNoRestore(true)
-                                        .SetVersion(NugetVersion.Value)
-                                        .SetInformationalVersion(GitVersionInfo?.InformationalVersion));
-                  });
+            _ => //_.DependsOn(CheckForbiddenWords)
```

Comment:
comments?

Reply:
reverted - was a shortcut to speed up manual testing